go-controller/pkg/clustermanager/userdefinednetwork/controller_test.go

```diff
@@ -1334,6 +1386,18 @@ func testNAD() *netv1.NetworkAttachmentDefinition {
 	}
 }
 
+func invalidTestNamespace(name string) *corev1.Namespace {
```

Comment:
super nit: I would define this function to use  `testNamespace` instead of the other way around.